webviz_config/containers/_syntax_highlighter.py

```diff
@@ -12,38 +12,25 @@ class SyntaxHighlighter(WebvizContainer):
 automatically detected.
 
 * `filename`: Path to a file containing the code to highlight.
-* `show_line_numbers`: If `True`, show line numbers. Default is `False`.
-* `start_number`: This only has effect if `show_line_numbers` is `True`. If
-                  given, line numbers will start at this value. Default is `1`.
 * `dark_theme`: If `True`, the code is shown with a dark theme. Default is
                 `False` giving a light theme.
 '''
 
-    def __init__(self, filename: Path, show_line_numbers: bool = False,
-                 start_number: int = 1, dark_theme: bool = False):
+    def __init__(self, filename: Path, dark_theme: bool = False):
 
         self.filename = filename
-        self.show_line_numbers = show_line_numbers
-        self.start_number = start_number
-        self.dark_theme = dark_theme
-
+        self.config = {'theme': 'dark'} if dark_theme else {'theme': 'light'}
         self.syntax_highlighter_id = 'syntax-highlighter-{}'.format(uuid4())
 
         with get_path(self.filename).open() as fh:
-            self.code = fh.read()
+            self.code = f'```{fh.read()}```'
```

Comment:
Perhaps this whole `with` block can be removed - and instead in `dcc.Markdown` use [`get_path(self.filename).read_text()`](https://github.com/equinor/webviz-config/blob/3da191e451870f0e5990d8a469419946e461e3d1/webviz_config/containers/_markdown.py#L114).